Makefile

```diff
@@ -1,5 +1,5 @@
 PLUGIN_NAME  := plymouth
-PLUGINS_PATH := $(DESTDIR)/lib/rc/plugins
+PLUGINS_PATH := $(DESTDIR)/usr/lib/rc/plugins
```

Comment:
This changes one form of breakage for another.

It doesn't respect https://www.gnu.org/prep/standards/html_node/Directory-Variables.html which should be used to define prefix / libdir. That is the only valid solution -- of course ultimately it should be ported to a real build system. :D